install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```
@@ -1,5 +1,41 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
+metadata:
+  name: machine-config-controller
+  namespace: openshift-machine-config-operator
+  labels:
+    k8s-app: machine-config-controller
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  groups:
+    - name: mcc-paused-pool-kubelet-ca
+      rules:
+        - alert: MachineConfigControllerPausedPoolKubeletCA
+          expr: |
+             sum by (pool) (last_over_time(machine_config_controller_paused_pool_kubelet_ca[5m])) > 0
```

as per the [alerting guidelines](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md), you should probably preserve the `namespace` label.

```suggestion
             sum by (namespace, pool) (last_over_time(machine_config_controller_paused_pool_kubelet_ca[5m])) > 0
```

---

I'm also not sure that `sum()` is the correct aggregation to use. `max()` is probably a better choice.

---

Thank you, I've preserved the namespace and switched from sum() to max()